src/Translator/composer.json

```
@@ -0,0 +1,51 @@
+{
+    "name": "symfony/ux-translator",
+    "type": "symfony-bundle",
+    "description": "Symfony Translator for JavaScript",
+    "keywords": [
+        "symfony-ux"
+    ],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Hugo Alliaume",
+            "email": "hugo@alliau.me"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "autoload": {
+        "psr-4": {
+            "Symfony\\UX\\Translator\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Symfony\\UX\\Translator\\Tests\\": "tests/"
+        }
+    },
+    "require": {
+        "php": ">=8.0",
+        "symfony/console": "^4.4|^5.0|^6.0",
+        "symfony/filesystem": "^5.4|^6.0",
+        "symfony/string": "^4.4|^5.0|^6.0",
+        "symfony/translation": "^4.4|^5.0|^6.0",
+        "twig/twig": "^2.0|^3.0"
+    },
+    "require-dev": {
+        "symfony/framework-bundle": "^4.4|^5.0|^6.0",
+        "symfony/phpunit-bridge": "^5.2|^6.0",
+        "symfony/twig-bundle": "^4.4|^5.0|^6.0",
+        "symfony/var-dumper": "^4.4|^5.0|^6.0"
```

```suggestion
        "symfony/var-dumper": "^5.4|^6.0"
```

Use this constraint for all of them - no need to support old stuff for a new package.

---

Done